apps/services/temporary-voter-registry-api/src/openApi.ts

```diff
@@ -1,8 +1,22 @@
+import { TemporaryVoterRegistryScope } from '@island.is/auth/scopes'
 import { DocumentBuilder } from '@nestjs/swagger'
+import { environment } from './environments'
 
 export const openApi = new DocumentBuilder()
   .setTitle('Temporary voter registry')
   .setDescription('This api contains access to the temporary voter registry.')
   .setVersion('1.0')
   .addTag('temporaryVoterRegistry')
+  .addOAuth2({
+    type: 'oauth2',
+    flows: {
+      authorizationCode: {
+        authorizationUrl: environment.swagger.authUrl,
+        tokenUrl: environment.swagger.tokenUrl,
```

Comment:
I would consider to link this to the actual IdP by using `environment.auth.issuer`.
That way. the OpenApi will be correct in each environment and easy to change the IdP host in the future.
```suggestion
        authorizationUrl: `${environment.auth.issuer}/connect/authorize`,
        tokenUrl: `${environment.auth.issuer}/connect/token`,
```

Reply:
Good idea, will change